library/src/main/java/dev/hichamboushaba/suspendactivityresult/Helpers.kt

```diff
@@ -90,4 +92,18 @@ suspend fun ActivityResultManager.openDocumentTree(startingLocation: Uri? = null
 @RequiresApi(19)
 suspend fun ActivityResultManager.createDocument(fileName: String): Uri? {
     return requestResult(CreateDocument(), fileName)
-}
\ No newline at end of file
+}
+
+/**
+ * see [StartIntentSenderForResult]
+ */
+suspend fun ActivityResultManager.takeActivityResult(intentSenderRequest: IntentSenderRequest): ActivityResult? {
+    return requestResult(StartIntentSenderForResult(), intentSenderRequest)
+}
+
+/**
+ * see [StartActivityForResult]
+ */
+suspend fun ActivityResultManager.takeStartActivityResult(intent: Intent): ActivityResult? {
```

Comment:
I think a name like `startActivityForResult` would be clearer on what the function does.

Reply:
fix